src/room.py

```diff
@@ -1,2 +1,12 @@
 # Implement a class to hold room information. This should have name and
-# description attributes.
\ No newline at end of file
+# description attributes.
+
+
+class Room:
```

Comment:
Nice; if this is working how you want it to, the next step will be to add functionality for an item to be added and removed from the room class.